aloscene/tensors/augmented_tensor.py

```diff
@@ -699,6 +699,17 @@ def as_tensor(self):
         n_tensor.__class__ = torch.Tensor
         n_tensor.rename_(None)
         return n_tensor
+    
+    def as_numpy(self, dtype=np.float16):
+        """Returns squeezed numpy array
+        
+        Parameters
+        ----------
+            dtype: (: np.dtype)
+                The output dtype. Default np.float16.
+        """
+        tensor = self
+        return np.squeeze(tensor.detach().cpu().numpy()).astype(dtype)
```

Comment:
As mentioned before, this method is not supposed to squeeze. squeeze can be done by the user itself after having called as_numpy()